src/components/Section/Section.tsx

```diff
@@ -0,0 +1,34 @@
+import { useStyles } from '@helpers/hooks/useStyles'
+import { FC, ReactNode } from 'react'
+
+import styles from './Section.module.scss'
+
+export interface SectionProps {
+  className?: string | string[]
+  children: ReactNode
+  space?: string
```

Comment:
to powinna być unia adekwatna do tego co w .scss

Reply:
kurcze, tak sie zajarałem tymi storiesami że są, że zapomniałem o tym ;/ poprawiam